### PR TITLE
[zh] Translate ambient/usage/troubleshoot-waypoint/

### PR DESCRIPTION
**Pull request by windsonsea**
## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/ambient/usage/troubleshoot-waypoint/index.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    {{< /text >}}
+
+    如果信息不够，您可以为 waypoint 代理启用调试日志：
+
+    {{< text bash >}}
+    $ istioctl pc log deploy/waypoint --level debug
+    {{< /text >}}
+
+1.  通过 `istioctl proxy-config` 命令检查 waypoint 的 Envoy 配置，
+    该命令显示与 Waypoint 相关的所有信息，如集群、端点、监听器、路由和密钥：
```
Comment (wilsonwu) [inline content/zh/docs/ambient/usage/troubleshoot-waypoint/index.md:113]:
```suggestion
    该命令显示与 waypoint 相关的所有信息，如集群、端点、监听器、路由和密钥：
```

File: content/zh/docs/ambient/usage/troubleshoot-waypoint/index.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+    {{< text bash >}}
+    $ istioctl proxy-config all deploy/waypoint
+    {{< /text >}}
+
+有关如何调试 Envoy 的更多信息，
+请参阅[深入了解 Envoy 配置](/zh/docs/ops/diagnostic-tools/proxy-cmd/#deep-dive-into-envoy-configuration)一节，
+因为 Waypoint 代理是基于 Envoy 的。
```
Comment (wilsonwu) [inline content/zh/docs/ambient/usage/troubleshoot-waypoint/index.md:121]:
```suggestion
因为 waypoint 代理是基于 Envoy 的。
```

Comment (windsonsea) [reply content/zh/docs/ambient/usage/troubleshoot-waypoint/index.md]: Fine, updated.

**Review by wilsonwu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**